node/core/dispute-coordinator/src/import.rs

```diff
@@ -488,7 +488,7 @@ impl ImportResult {
 		for (index, sig) in approval_votes.into_iter() {
 			debug_assert!(
 				{
-					let pub_key = &env.session_info().validators[index.0 as usize];
+					let pub_key = &env.session_info().validators.get(index).expect("<write an actual proof here; qed>");
```

Comment:
:sweat_smile: I didn't mean `write an actual proof here` quite literally. If you grep for [`qed`](https://cs.github.com/paritytech/polkadot?q=qed), you'll see what kind of proof I meant. In this case it would be something like:
```suggestion
					let pub_key = &env.session_info().validators.get(index).expect("indices are validated by approval-voting subsystem; qed");
```